### DISCUSSION TITLE
Validation problem

### DISCUSSION THREAD

**Discussion by hosseinnegahban** [mdolab/dafoam#274] [Bug]
I run the subsonic NACA 0012 case in tutorials  for U=51m/s (M=0.15) without any modification in runScript and I only  use solveCL, but the problem is that the drag coefficient is very different from the results obtained by this website https://turbmodels.larc.nasa.gov/naca0012_val.html  but the CL is fine.

**Comment by friedenhe** [comment]
Make sure you are using the same condition and mesh as the NASA website
